pages/api/v1/reset-workspace.ts

```diff
@@ -0,0 +1,35 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { prismaClient } from "../../../lib/prisma-client";
+import { getToken } from "next-auth/jwt";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  console.log("/reset-workspace", req.body);
+
+  const token = await getToken({
+    req: req,
+  });
+  // console.log("gSSP token", token);
+
+  if (!token?.sub) {
+    throw new Error("No session");
+  }
+
+  const prisma = prismaClient;
```

Comment:
I would just reference `prismaClient` directly, no need to make a separate variable.

`prismaClient.options..`